prisma-fmt/tests/code_actions/scenarios/multi_schema_add_to_existing_schemas/schema.prisma

```diff
@@ -0,0 +1,17 @@
+generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["multiSchema"]
+}
+
+datasource db {
+    provider     = "postgresql"
+    url          = env("DATABASE_URL")
+    schemas      = ["a", "b"]
+    relationMode = "prisma"
+}
+
+model A {
+    id Int @id
+
+    @@schema("base")
+}
```

Comment:
It looks to me that we could replace this test for the following simple case (based on the name of the test)
```prisma
generator client {
    provider        = "prisma-client-js"
    previewFeatures = ["multiSchema"]
}

datasource db {
    provider     = "postgresql"
    url          = env("DATABASE_URL")
    schemas      = ["a"]
    relationMode = "prisma"
}

model A {
    id Int @id
    @@schema("a")
}

model B {
    id Int @id
    @@schema("b")
}
```

where ` schemas      = ["a"]` already exists and is missing one schema.

the current test could be moved as an additional test like `multi_schema_fix_existing_schemas/schema.prisma`?